build.gradle

```diff
@@ -35,7 +35,7 @@ apply plugin: 'java-library'
 apply plugin: 'groovy'
 apply plugin: 'com.palantir.external-publish-jar'
 
-sourceCompatibility = 1.8
+sourceCompatibility = 11
```

Comment:
I don't think we want to bumpt he JDK version. Otherwise we won't be able to use it in Iceberg

Reply:
My bad, let me revert. Good catch !